routes/cmip6.py

```diff
@@ -0,0 +1,187 @@
+    """
+    Make an async request for CMIP6 monthly data for a range of models, scenarios, and years at a specified point
+
+    Args:
+        lat (float): latitude
+        lon (float): longitude
+        var_coord (int): variable coordinate from dim_encoding, if specified
+
+    Returns:
+        list of data results from each of historical and future data at a specified point
+    """
+
+    # We must use EPSG:4326 for the CMIP6 monthly coverage to match the coverage projection
+    wcs_str = generate_wcs_getcov_str(
+        lon,
+        lat,
+        cov_id=cmip6_monthly_coverage_id,
+        projection="EPSG:4326",
+        var_coord=var_coord,
+    )
+
+    # Generate the URL for the WCS query
+    url = generate_wcs_query_url(wcs_str)
+
+    # Fetch the data
+    point_data_list = await fetch_data([url])
+
+    return point_data_list
+
+
+def package_cmip6_monthly_data(point_data_list, var_id=None):
+    """
+    Package the CMIP6 monthly values into human-readable JSON format
+
+    Args:
+        point_data_list (list): nested list of data from Rasdaman WCPS query
+
+    Returns:
+        di (dict): dictionary mirroring structure of nested list with keys derived from dim_encodings global variable
+    """
+    di = dict()
+
+    # Nest point_data_list one level deeper if var_id is specified.
+    # This keeps the nesting level the same for all cases.
+    if var_id != None:
+        point_data_list = [point_data_list]
+
+    for var_coord, var_li in enumerate(point_data_list):
+        if var_id != None:
+            varname = var_id
+        else:
+            varname = dim_encodings["varname"][var_coord]
+        for mi, model_li in enumerate(var_li):
+            model = dim_encodings["model"][mi]
+            if model not in di:
+                di[model] = dict()
+            for si, scenario_li in enumerate(model_li):
+                scenario = dim_encodings["scenario"][si]
+                if scenario not in di[model]:
+                    di[model][scenario] = dict()
+
+                # Create an array of every month since January 1950 in the format "YYYY-MM"
+                months = [
+                    f"{year}-{str(month).zfill(2)}"
+                    for year in range(1950, 2100 + 1)
+                    for month in range(1, 13)
+                ]
+
+                for soi, value in enumerate(scenario_li):
+                    month = months[soi]
+                    if month not in di[model][scenario]:
+                        di[model][scenario][month] = dict()
+
+                    # The "ts" variable is still in Kelvin in the Rasdaman coverage.
+                    # Convert this to Celsius. All other temperature variables have
+                    # already been converted to Celsius before Rasdaman import.
+                    if varname == "ts" and float(value) != -9999:
+                        value = float(value) - 273.15
+
+                    # Evaporation has very tiny values.
```

Comment:
Also appreciate this comment, I was wondering what was up with those values when I saw them in the response.